share/models/change.py

```diff
@@ -271,3 +242,66 @@ def _resolve_change(self):
             else:
                 change[k] = v
         return change
+
+    def _merge_objects(self, from_obj, into_obj):
+        from share.models import ShareObject
+
+        # TODO less hacky way of expressing what can be merged
+        MERGEABLE_MODELS = {'AbstractCreativeWork', 'AbstractAgent', 'Award', 'AbstractWorkRelation', 'AbstractAgentRelation', 'AbstractAgentWorkRelation', 'ThroughTags', 'ThroughSubjects', 'ThroughContributor', 'ThroughAwards'}
+        concrete_model = from_obj._meta.concrete_model
+        if concrete_model.__name__ not in MERGEABLE_MODELS:
+            raise InvalidMergeError('Invalid model for merging: %s' % concrete_model)
```

Comment:
Makes sense. I definitely wrote these like they were log messages and made the minimal possible change when I realized that was wrong.